src/modules/endpoint/DatabaseConnector.js

```diff
@@ -73,11 +83,15 @@ const mapToObject = (map: Map<string, string>) => {
 }
 
 class DatabaseConnector {
-  getContentPath (key: string, context: DatabaseContext): string {
+  getContentPath (key: string, context: DatabaseContext = null): string {
```

Comment:
Should probably be a `context: ?DatabaseContext = null`.

Reply:
A more clear syntax would be: `(key: string, context?: DatabaseContext)`
Then you can just omit the context